openlane/scripts/yosys/json_header.tcl

```diff
@@ -13,12 +13,11 @@
 # limitations under the License.
 yosys -import
 source $::env(SCRIPTS_DIR)/yosys/common.tcl
-set vtop $::env(DESIGN_NAME)
 
 source $::env(_DEPS_SCRIPT)
 
-read_verilog_files $vtop
-hierarchy -check -top $vtop
-yosys rename -top $vtop
+yosys_ol::read_verilog_files $::env(DESIGN_NAME)
+hierarchy -check -top $::env(DESIGN_NAME) -nokeep_prints -nokeep_asserts
```

Comment:
what are `-nokeep_prints` and `-nokeep_asserts` for?

Reply:
Allegedly, they remove print and assert cells that cause issues in OpenROAD. Practically, nothing, but they don't hurt either.